addons/web/static/src/js/views/form_widgets.js

```diff
@@ -1213,6 +1213,7 @@ var FieldBinaryFile = FieldBinary.extend({
     },
     on_clear: function() {
         this._super.apply(this, arguments);
+        this.$el.find('input[type="file"]').val('');
         this.$el.find('input').eq(0).val('');
```

Comment:
ok, I use  <code>this.$el.find('input').val('')</code> for reset all input. thank you,